src/test/rustdoc-ui/intra-link-prim-conflict.rs

```diff
@@ -14,7 +14,7 @@
 //~| HELP to link to the builtin type
```

Comment:
```suggestion
//~| HELP to link to the built-in type
```
there are a number of occurrences of this, just flagging the first one, uncertain about the proposed correctness but the surface is small

Reply:
It doesn't make sense to change this. This a test that is pattern matching on an error message. If you want to change the error message to use a different spelling, please open a PR changing the spelling in the source.